_posts/2016-11-01-solidity-0.4.4-release-announcement.md

```diff
@@ -7,7 +7,7 @@ author: Solidity Team
 category: Releases
 ---
 
-This is a bugfix release that fixes a storage corruption that appears when multiple variables are stored in the same slot ([details](https://solidity.ethereum.org/2016/11/01/security-alert-solidity-variables-can-be-overwritten-in-storage/)).
+This is a bugfix release that fixes a storage corruption that appears when multiple variables are stored in the same slot ([details](https://blog.soliditylang.org/2016/11/01/security-alert-solidity-variables-can-overwritten-storage/)).
```

Comment:
This was actually a broken link, as the original one was https://blog.ethereum.org/2016/11/01/security-alert-solidity-variables-can-overwritten-storage/ and we kept the filename.

Reply:
Since this was auto generated from the release notes on GitHub, the link is broken there too.